src/management-system-v2/lib/process-export/copy-process-image.ts

```diff
@@ -8,17 +8,43 @@ import { BPMNCanvasRef } from '@/components/bpmn-canvas';
  *
  * @param modeler the modeler to copy the process image from
  */
-export async function copyProcessImage(modeler: BPMNCanvasRef) {
-  let xml = await modeler.getXML();
+
+export async function copyProcessImage_(modeler: any) {
```

Comment:
Could't this be one function? The only difference is getting the xml and the canvas based on the Input being either a bpmn-js modeler or our BPMNCanvasRef. Everything else is the same.